railties/lib/rails/generators/rails/app/templates/Gemfile

```diff
@@ -25,6 +25,10 @@ git_source(:github) { |repo| "https://github.com/#{repo}.git" }
 
 <%- end -%>
 <% if RUBY_ENGINE == 'ruby' -%>
+
+# Reduces boot times through caching; configured in config/boot.rb
+gem 'bootsnap', '~> 1.0.0', platforms: [:mri]
```

Comment:
@burke could bootsnap allow installation on 2.2, and just operate in a degraded mode? AIUI path caching would still work -- it's the iseq cache that is strictly 2.3+, right?